tools/c7n_tencentcloud/tests/test_tc_client.py

```diff
@@ -2,266 +2,151 @@
 # SPDX-License-Identifier: Apache-2.0
 
 from datetime import datetime
+import jmespath
 import pytest
 import socket
 from retrying import RetryError
 from c7n_tencentcloud.utils import PageMethod
 from c7n.exceptions import PolicyExecutionError
-from c7n_tencentcloud.client import Client
 from tencentcloud.common.abstract_client import AbstractClient
 from tencentcloud.common.exception.tencent_cloud_sdk_exception import TencentCloudSDKException
 
 
-def test_call_client(session):
-    endpoint = "region.tencentcloudapi.com"
-    service = "region"
-    version = "2022-06-27"
-    region = "ap-singapore"
-    action = "DescribeProducts"
-    param = {}
-    cli = session.client(endpoint, service, version, region)
-    with pytest.raises(TencentCloudSDKException):
-        cli.execute_query(action, param)
+class TestClient:
+    @pytest.fixture
+    def simple_client(self, session):
+        return session.client("region.tencentcloudapi.com", "region", "2022-06-27", "ap-shanghai")
 
+    @pytest.mark.vcr
+    def test_query_simple(self, simple_client):
+        # cli = session.client("region.tencentcloudapi.com", "region", "2022-06-27", "ap-shanghai")
+        action = "DescribeProducts"
+        jsonpath = "Response.Products[]"
+        resp = simple_client.execute_query(action, {})
+        data = jmespath.search(jsonpath, resp)
+        assert data
```

Comment:
can this assert additionally some value in the data?

Reply:
the original idea is just checking whether there is a value.
sure, will add some value next time